docs/excel/excel-add-ins-ranges-unbounded.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Read or write to an unbounded range using the Excel JavaScript API
+description: 'Learn how to use the Excel JavaScript API to read or write to an unbounded range.'
+ms.date: 04/02/2021
+localization_priority: Normal
+---
+
+# Read or write to an unbounded range using the Excel JavaScript API
+
+This article describes how to read and write to an unbounded range with the Excel JavaScript API. For the complete list of properties and methods that the `Range` object supports, see the [Excel.Range class](/javascript/api/excel/excel.range).
+
+## Read an unbounded range
+
+An unbounded range address is a range address that specifies either entire columns or entire rows. For example:
+
+- Range addresses comprised of entire columns:<ul><li>`C:C`</li><li>`A:F`</li></ul>
+- Range addresses comprised of entire rows:<ul><li>`2:2`</li><li>`1:4`</li></ul>
```

Comment:
This definition should be moved up a level, in the H1 description paragraph. It helps set context for the entire article, not just reads.